### DISCUSSION TITLE
XrayR v0.8.7

### DISCUSSION THREAD

**Discussion by pocketW** [XrayR-project/XrayR#148] [Announcements]
# XrayR v0.8.7

## Feat
* 重构全局设备限制功能，现在能直接读取远程面板下发的设备限制数量，并且采用缓存机制，降低延迟。请查看[配置文档](https://xrayr-project.github.io/XrayR-doc/gong-neng-shuo-ming/devicelimit.html)修改配置 @thank243 @pocketW 
* 支持V2board最新API。
v2board (dev branch) 版本 > [1.6.1](https://github.com/v2board/v2board/releases/tag/1.6.1)请选择 `PanelType: "NewV2board"`。其他旧版本选择 `PanelType: "V2board"`，**注意老版本API将于 2023.6.1后移除**。

## Fix
* 修复http模式无法申请证书问题 @thank243 
* 修复SSpanl结点信息汇报问题 @amin550 
* 修复Typo

## Chore
* 升级Xray-core 至1.6.5
* 升级其他依赖

## Note
XrayR已完成针对Shadowsocks-2022的适配工作，等待各大面板支持Shadowsocks-2022的信息下发。

## New Contributors
* @Github-Aiko made their first contribution in https://github.com/XrayR-project/XrayR/pull/105
* @amin550 made their first contribution in https://github.com/XrayR-project/XrayR/pull/118

**Full Changelog**: https://github.com/XrayR-project/XrayR/compare/v0.8.6...v0.8.7

<hr /><em>This discussion was created from the release <a href='https://github.com/XrayR-project/XrayR/releases/tag/v0.8.7'>XrayR v0.8.7</a>.</em>

**Comment by Septrum101** [comment]


**Reply by pocketW** [reply]
Hi @thank243 Can you give me a link to the error? or you can directly fix it in the [XrayR-peoject/XrayR-doc](https://github.com/XrayR-project/XrayR-doc).

**Reply by pocketW** [reply]
Do you mean ?
> 可在ControllerConfig 中配置如下项。

The config of GlobalLimit is a sub-config under the ControllerConfig setting.
https://github.com/XrayR-project/XrayR/blob/0bea22e1454ae489ae03865c9dbca0a588700115/service/controller/config.go#L22

**Reply by Septrum101** [reply]
I'm sorry. I recognize ControllerConfig as ConnectionConfig. LOL
